.github/workflows/stable-build.yml

```diff
@@ -0,0 +1,66 @@
+name: Stable blocks publish to stable tag
```

Comment:
This is a new file for stable. Is the plan to rely on this automation for production releases going forward?

Reply:
I'd hope so. at least based on the ticket requirements